controllers/authpolicy_authconfig.go

```diff
@@ -235,8 +235,8 @@ func getAffectedPolicies(t *kuadrantgatewayapi.Topology, ap *api.AuthPolicy) []k
 	return affectedPolicies
 }
 
-// authConfigName returns the name of Authorino AuthConfig CR.
-func authConfigName(apKey client.ObjectKey) string {
+// AuthConfigName returns the name of Authorino AuthConfig CR.
+func AuthConfigName(apKey client.ObjectKey) string {
```

Comment:
Are we planning on moving this out to another package in another iteration? Is that the reasoning for exporting the function?

Reply:
my bad. I initially moved the tests to another package. Fix it.

Reply:
done